main.py

```diff
@@ -1,10 +1,13 @@
-from flask import Flask, request, jsonify
+from flask import Flask, request, render_template, jsonify
 from flask_restful import Api, Resource, abort
+import requests
 import argparse
 from halyard_utils import add_flag
-from instance.list_nodes import list_nodes, get_node
 from image.create_base_image import create_base_image
 from instance.create_or_restore_instance import create_or_restore_instance, create_instance
+from instance.node_manager import list_nodes, get_node, delete_node
+from image.image_manager import list_images, get_image, delete_image
```

Comment:
nit: will the create part be folded into the manager?  Just seems a little off that list, get, delete are all part of the manager, but not the create.

Reply:
I was thinking about that, but it's just that the create is mucho more elaborate on the things it does, involving multiple scripts, defining specific behavior for cuttlefish and managing the os directly while the manager functions are moslty simple calls to the libcloud API to interact with gcloud elements

Reply:
fine to leave as is, but in my mind, the create call of the manager will just call the existing methods for creating an instance or image. It's a aesthetics thing more than anything else.

Reply:
I agree with Po: create should be part of the manager interface along with list, get and delete. Whether it's a more complicated action or not is just an implementation detail that shouldn't affect the interface.

Reply:
Ok then better I'll merge those functions into the manager